source/frontend/src/features/mapSideBar/acquisition/add/AcquisitionPropertiesSubForm.tsx

```diff
@@ -46,14 +50,48 @@ export const AcquisitionPropertiesSubForm: React.FunctionComponent<
                             ? AddressForm.fromBcaAddress(bcaSummary?.address)
                             : undefined;
                         }
-                        if (
-                          values.properties?.length === 0 &&
-                          index === 0 &&
-                          formProperty.regionName !== 'Cannot determine'
-                        ) {
-                          formikProps.setFieldValue(`region`, formProperty.region);
+
+                        if (formProperty.apiId && (await confirmBeforeAdd(formProperty.apiId))) {
+                          // Require user confirmation before adding property to file
+                          setModalContent({
+                            variant: 'warning',
+                            title: 'User Override Required',
+                            message: (
+                              <>
+                                <p>
+                                  This property has already been added to one or more acquisition
+                                  files.
+                                </p>
+                                <p>Do you want to acknowledge and proceed?</p>
+                              </>
+                            ),
+                            okButtonText: 'Yes',
+                            cancelButtonText: 'No',
+                            handleOk: () => {
```

Comment:
Hmm, I don't see anything in the AC that reflects why this is here, what am I missing?